site/content/docs/manifest/scheduled-job.ja.md

```diff
@@ -97,6 +103,22 @@ Dockerfile からコンテナイメージをビルドする代わりに、既存
 <span class="parent-field">image.</span><a id="image-labels" href="#image-labels" class="field">`labels`</a><span class="type">Map</span>  
 コンテナに付与したい [Docker ラベル](https://docs.docker.com/config/labels-custom-metadata/)を key/value の Map で指定できます。これは任意設定項目です。
 
+<span class="parent-field">image.</span><a id="image-depends-on" href="#image-depends-on" class="field">`depends_on`</a> <span class="type">Map</span>  
+コンテナに追加する [Container Dependencies](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDependency.html) の任意の key/value の Map。Map の key はコンテナ名で、value は依存関係を表す値 (依存条件) として `start`、`healthy`、`complete`、`success` のいずれかを指定できます。なお、Essential コンテナに `complete` や `success` の依存条件を指定することはできません。
```

Comment:
Can we update here as we update in https://github.com/aws/copilot-cli/pull/2710/files#diff-1bf141fd0bc7bce3457ac661e591f68f063d9d5bbf109699c4fe9d15b1081fa6R47?

Reply:
ok